src/components/Card/HashCard/index.tsx

```diff
@@ -38,6 +39,7 @@ export default ({
   const newAddr = useNewAddr(hash)
   const deprecatedAddr = useDeprecatedAddr(hash)
   const counterpartAddr = newAddr === hash ? deprecatedAddr : newAddr
+  const alias = useDASAccount(isTx ? '' : hash)
```

Comment:
This component is also used on the block page(https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/pages/BlockDetail/index.tsx#L95) for block hash and on Simple UDT page(https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/pages/SimpleUDT/index.tsx#L125) for `udt title`

Reply:
Here is my oversight, change pushed 🌚